dali/pipeline/data/tensor_test.cc

```diff
@@ -436,7 +436,7 @@ TYPED_TEST(TensorTest, TestCustomAlloc) {
   std::function<void(void *)> deleter;
   if (std::is_same<TypeParam, CPUBackend>::value) {
     alloc_func = [&](size_t bytes) {
-      return new uint8_t[bytes];
+      return malloc(sizeof(uint8_t) * bytes);
```

Comment:
Why malloc instead of new?

Reply:
Because we use plain `free` in L442. I can use `delete []` there instead.